packages/api/src/schema/schema.yaml

```diff
@@ -1674,6 +1677,14 @@ components:
               $ref: "#/components/schemas/task/properties/params/properties/upload"
             export:
               $ref: "#/components/schemas/export-task-params"
+            delete:
+              type: object
+              additionalProperties: false
+              description: Parameters for the delete task
+              properties:
+                id:
+                  type: string
+                  description: ID of the asset to delete
```

Comment:
Can you expand this description explaining that this field is not necessary and not used, kept here only to avoid an empty object?